### PR TITLE
Adds the syndicate Booze-O-Mat (Bruise-O-Mat) to nukie planet

### LINKED ISSUE #31586: Nukies can't interact with the booze-o-mat in the nukie planet
## Description

Missing access, so there should be a unique entity that doesn't require access. Maybe even a unique sprite/name/description?

**Screenshots**

image

### PR DESCRIPTION
**Pull request by ArtisticRoomba**
## About the PR
Adds a syndicate Booze-O-Mat to the nukie planet, aptly named the Bruise-O-Mat.

## Why / Balance

Right now, nukies have to take their extremely important and totally-not-often-wasted time to hack the Booze-O-Mat on the nukie outpost, because it has station-relevant accesses. Hydration and (un)responsible drinking is top priority. This PR proposes a solution: the syndicate Booze-O-Mat, named the Bruise-O-Mat.

This new machine adds to the funny atmosphere of most Syndicate-aligned machines. Now nukies will no longer have to hack the Booze-O-Mat to get at their precious alcohol.

This PR closes #31586.

I can't give the machine a custom inventory because skill issue. I think it has enough charm already.

## Technical details
YAML and FTL editing lmao.
New machine `VendingMachineBoozeSyndicate` aka Bruise-O-Mat.
New file bruiseomat.ftl for storing the absurd amount of advertising and thankyou lines.
Relevant edits to advertisements.yml and goodbyes.yml.
New textures for the Bruise-O-Mat.
Replacement of the Booze-O-Mat for the Bruise-O-Mat on the nukie planet.

## Media

Content Client_xN
![Content Client_alLsYB4Fbs
9KsrqmIs](https://github.com/user-attachments/assets/be1c6c95-ff03-4b67-802a-445dd7c15fe7)
Content Client_8880fCClb5
Content Client_6Z86OPtnT9
Content Client_Eo4b1bcexe
Content Client_alLsYB4Fbs

https://github.com/user-attachments/assets/b93100e0-ec4f-4889-99d8-1a3248a72b5e

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- add: The Bruise-O-Mat alcohol vendor has been added to the nukie outpost, for all your pre-op drinking needs. Seems to have developed a witty personality, too...

### CODE REVIEW

File: Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml [hunk 1]
```diff
@@ -435,6 +435,23 @@
     - DrinkShaker
     - BorgDropper
 
+# service modules
```
Comment (ArtisticRoomba) [inline Resources/Prototypes/Entities/Objects/Specific/Robotics/borg_modules.yml:438]: I know I fucked up here. I'll undo changes when I get home. Moron.

**Comment by ImRenQ** [comment]
funny lines

**Comment by ArtisticRoomba** [comment]
please dont look at git

**Comment by Emisse** [comment]
Merging from out of country

**Pull request merged**